capuccino/capuccino_py.py

```diff
@@ -0,0 +1,28 @@
+from math import *
+from espresso.espresso_py import espresso
+
+class capuccino(espresso):
+    def __init__(self, c_coffee, c_sugar, c_water,  milk):
+        super().__init__(coffee = c_coffee, sugar = c_sugar, water = c_water)
```

Comment:
When calling methods you should not use spaces before and after = 
https://www.python.org/dev/peps/pep-0008/#other-recommendations

Reply:
amended